clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsClusterProvider.java

```diff
@@ -36,4 +51,52 @@ public EcsClusterProvider(Cache cacheView) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterCacheClient.getAll();
   }
+
+  // TODO include[] input of Describe Cluster is not part of this implementation, need to implement
+  // in the future.
```

Comment:
Not a comment but rather curious, what inputs did you have in mind?

Reply:
Good question. So, Describe Cluster API responds with ATTACHMENTS | SETTINGS | STATISTICS | TAGS only when they are included in the include[] input. e.g. if you want description of TestCluster with it's SETTINGS then you would send TestCluster inside` Clusters["TestCluster"] `and include settings like this `include["SETTINGS"].`

Reply:
Got it! That's awesome! Do you see a need for any of those at the moment?

Reply:
No, we do not need them for now.